sql-to-dbsp-compiler/lib/sqllib/src/operators.rs

```diff
@@ -86,22 +86,144 @@ for_all_compare!(gte, bool);
 #[inline(always)]
 fn plus<T>(left: T, right: T) -> T
 where
-    T: Add<Output = T>,
+    T: CheckedAdd,
 {
+    left.checked_add(&right)
+        .expect("attempt to add with overflow")
+}
+
+for_all_int_operator!(plus);
+
+pub fn plus_f_f(left: F32, right: F32) -> F32 {
```

Comment:
you may want to generate these with a different macro, which can never fail

Reply:
I thought about it but then decided to do it by hand, as it'd be very similar to `some_operator!` and `some_existing_operator!`, and could cause confusion in future.